### PR TITLE
Emit input whenever WYSIWYG content is updated

### LINKED ISSUE #13798: Editing href in code view at a WYSIWYG field does not mark the field as dirty
### Describe the Bug

When editing a existing content with a wysiwyg field which has a link inside it, if you edit the href attribute in the code view (to change part of the url for example), and then you save the code tab, the content save button is not highlighted, and so the changes cannot be saved. 

_Note_: I found some issue where this problem was addressed but the changes where removed some months after. I've not found the explanation about why (and I don't know Vue enough to propose a theory)

The original issue: #9978 
The issue that removed the check: #10548 "Fix wysiwyg dirty check"

### To Reproduce

1. Fresh Directus installation in Ubuntu 20.04 (local)
2. Create "content" Collection, add only a wysiwyg field with default configuration.
3. Go to Content section and "Create Item"
4. Fill the field with some text and a link. Save.
5. Edit the content you just created, get into "Edit source code"
6. change part of the link, save the code
7. The save button on the content keeps being inactive

### Errors Shown

No errors shown anywhere (interface, JS console)

### What version of Directus are you using?

9.12.1

### What version of Node.js are you using?

16.15.1

### What database are you using?

MySQL 8.0.29

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

### PR DESCRIPTION
**Pull request by licitdev**
## Description

Fixes #13798 where the updated input was not emitted.

## Type of Change

- [x] Bugfix
- [ ] New Feature
- [ ] Refactor / codestyle updates
- [ ] Other, please describe:

## Requirements Checklist

- [ ] New / updated tests are included
- [x] All tests are passing locally
- [x] Performed a self-review of the submitted code

If adding a new feature:

- [ ] Documentation was added/updated

### CODE REVIEW

**Review by rijkvanzanten — COMMENTED** [review]
Let's make sure this doesn't cause another "are you sure you want to discard the changes" infinite loop like we've had before on the wysiwyg

**Comment by licitdev** [comment]
@rijkvanzanten Thanks! Missed out on this issue. 🙏🏼 Referred to https://github.com/directus/directus/pull/12936/files and refactored such that only the soft limit count is updated on init. The initial `@set-content` will also not emit as the observer is only set up after the `@focus` event is fired.

**Review by rijkvanzanten — APPROVED** [review]

File: app/src/interfaces/input-rich-text-html/input-rich-text-html.vue [hunk 1]
```diff
@@ -421,14 +422,21 @@ export default defineComponent({
...
+
 		function setupContentWatcher() {
 			if (observer) return;
 
 			const iframeContents = editorRef.value.contentWindow.document.getElementById('tinymce');
 
 			observer = new MutationObserver((_mutations) => {
-				count.value = iframeContents?.textContent?.replace('\n', '')?.length ?? 0;
-				emit('input', editorRef.value.getContent() ? editorRef.value.getContent() : null);
+				contentUpdated();
```
Comment (joselcvarela) [inline app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:439]: Hey @licitdev 
Someone reported that pasting code in WYSIWYG inside translations duplicates entries.
Can you give a look? 
I tested it and it seems it comes from here. I think adding a `debounce` will solve it but I wonder if we can filter the proper mutation using `_mutations` 🤔

Comment (licitdev) [reply app/src/interfaces/input-rich-text-html/input-rich-text-html.vue:439]: Thanks! Can confirm the issue when pasting content, there's indeed multiple firing of event. There's this TinyMCE [issue](https://github.com/tinymce/tinymce/issues/4471). Posted the issue at #14215.

**Pull request merged**